src/TypeSortedCollections.jl

```diff
@@ -220,6 +220,22 @@ end
     end
 end
 
+@generated function Base.any(f::F, tsc::TypeSortedCollection{<:Any, N}) where {F, N}
+    N == 0 && return false
```

Comment:
I know some people do this and it works right now, but AFAIK the proper, documented way is to always return quoted expressions in a generated function, so I'd prefer `return :(false)`.

Reply:
I actually tried that yesterday and discovered that for `AbstractFloat`s and `Integer`s, `@eval :($x) === x` (i.e., `:(true) === true` and `:(false) === false` at the very least). But I can use `quote false end` here to be sure.

Reply:
Ah yes. Fine as is then.